src/components/Modal/Auth/AuthInputs.tsx

```
@@ -0,0 +1,19 @@
+import { authModalState } from "@/src/atoms/authModalAtoms";
+import { FormC } from "@/src/styles/Modal.styled";
+import React from "react";
+import { useRecoilValue } from "recoil";
+import Login from "./Login";
+import SignUp from "./SignUp";
+
+type AuthInputsProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)